lib/cloudinary/api.rb

```diff
@@ -189,6 +189,24 @@ def self.resources_by_asset_ids(asset_ids, options={})
     call_api(:get, uri, params, options)
   end
 
+  # Returns all assets stored directly in a specified asset folder, regardless of the public ID paths of those assets.
+  #
+  # @param [Object] asset_ids The requested asset IDs.
```

Comment:
The first parameter is actually `asset_folder` and it is a string

Reply:
This has been updated.